main/config/settings/dev.yml

```diff
@@ -0,0 +1,125 @@
+# Target AWS region, defaults to us-east-1 if not provided.  Note thet some global system components
```

Comment:
should we get rid of these comments?

Reply:
Leaving comments for now as they are acting as documentations.